assets/js/src/core/components/editor-tabs/editor-tabs.tsx

```diff
@@ -22,6 +22,9 @@ import { IconWrapper } from '@Pimcore/components/editor-tabs/editor-tabs.icon-wr
 import { IconButton } from '../icon-button/icon-button'
 import { useElementContext } from '@Pimcore/modules/element/hooks/use-element-context'
 import useElementResize from '@Pimcore/utils/hooks/use-element-resize'
+import { useElementDraft } from '@Pimcore/modules/element/hooks/use-element-draft'
```

Comment:
Should we move the "editor-tabs" component somewhere in the `@Pimcore/modules/element/editor` namespace? In my opinion this is not a component anymore but a module (also before your a PR). Accessing files from the modules folder from within the components folder doesn't sound like a good idea. They components should only be view components without logic :)

Reply:
yeah sounds like a good point - i moved it to `modules/element/editor/tabs/editor-tabs`

Reply:
we would need to move the whole folder, not only the single file:
https://github.com/pimcore/studio-ui-bundle/tree/eefd1e6ea056eaaeeac3f0fbf7bf6c6f5e8f5e88/assets/js/src/core/components/editor-tabs